deploy/crd.yaml

```diff
@@ -207,10 +210,13 @@ spec:
                         type: string
                       credentialsSecret:
                         type: string
+                      endpointUrl:
```

Comment:
@nmarukovich please update cr.yaml and add test for this PR

Reply:
7423bf4f24e5228d4a7a89f170abfff1c8d3f20e fixed.